fastlane.gemspec

```diff
@@ -113,6 +113,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('google-cloud-storage', '~> 1.31') # Access Google Cloud Storage for match
   spec.add_dependency('emoji_regex', '>= 0.1', '< 4.0') # Used to scan for Emoji in the changelog
   spec.add_dependency('aws-sdk-s3', '~> 1.0') # Used for S3 storage in fastlane match
+  spec.add_dependency('aws-sdk-secretsmanager', '~> 1.0') # Used for Secrets Manager in fastlane match
```

Comment:
Is there an easy way to add support for the AWS Secret manager without having to add a new dependency to fastlane? We try our best to minimise adding new dependencies at the moment.

Reply:
I don't think so. Do you have any ideas?

Reply:
I could make manual http calls or use the shell (requires user to have the AWS SDK installed). I don't feel that it is a good way to do it though. 
Another way would be to somehow add support for plugins extending the storage backends - i actually think this would be the best way right after adding the dependency. 
Please do share your thoughts on this as i would like to move this forward.